groceries/templates/items/index.html

```diff
@@ -5,10 +5,20 @@ <h1>{% block title %}{{ items|length }} Items{% endblock %}</h1>
 {% endblock %}
 
 {% block content %}
+{#  <div>    <p>Here is my category_list: {{ category_list }}</p></div>#}
```

Comment:
I think these comments and extra whitespace are left over from some experimenting and can be removed